tests/test_cli_opts.py

```diff
@@ -63,3 +63,216 @@ def test_cli_prcellgid():
     )
     assert (test_folder_path / "62798_py_Neuron_t0.0.nrndat").is_file()
     assert (test_folder_path / "62798_py_Neuron_t100.0.nrndat").is_file()
+
+
+@pytest.mark.skipif(
+    not os.environ.get("NEURODAMUS_NEOCORTEX_ROOT"),
+    reason="Test requires loading a neocortex model to run")
+def test_cli_disable_reports():
+    test_folder = tempfile.TemporaryDirectory("cli-test-disable-reports")  # auto removed
+    test_folder_path = Path(test_folder.name)
+    for f in ("node_sets.json", "circuit_config.json", "input.dat", "simulation_config_mini.json"):
+        shutil.copy(SIM_DIR / f, test_folder_path)
+
+    with open(SIM_DIR / CONFIG_FILE_MINI, "r") as f:
+        sim_config_data = json.load(f)
+
+    subprocess.run(
+        ["neurodamus", CONFIG_FILE_MINI, "--disable-reports"],
+        check=True,
+        cwd=test_folder_path
+    )
+    # Spikes are present even if we disable reports
+    assert (test_folder_path / sim_config_data["output"]["output_dir"] / "out.h5").is_file()
+    for report in sim_config_data["reports"].keys():
+        report_path = test_folder_path / sim_config_data["output"]["output_dir"] / (report + ".h5")
+        assert not report_path.is_file(), f"File '{report_path}' should NOT exist."
+
+    subprocess.run(
+        ["neurodamus", CONFIG_FILE_MINI],
+        check=True,
+        cwd=test_folder_path
+    )
+    assert (test_folder_path / sim_config_data["output"]["output_dir"] / "out.h5").is_file()
+    for report in sim_config_data["reports"].keys():
+        report_path = test_folder_path / sim_config_data["output"]["output_dir"] / (report + ".h5")
+        assert report_path.is_file(), f"File '{report_path}' not found."
+
+
+@pytest.mark.skipif(
+    not os.environ.get("NEURODAMUS_NEOCORTEX_ROOT"),
+    reason="Test requires loading a neocortex model to run")
+def test_cli_keep_build():
+    with open(SIM_DIR / CONFIG_FILE_MINI, "r") as f:
+        sim_config_data = json.load(f)
+        sim_config_data["target_simulator"] = "CORENEURON"
+        sim_config_data["output"]["output_dir"] = "output_keep_build"
+
+    test_folder = tempfile.TemporaryDirectory("cli-test-keep-build")  # auto removed
+    test_folder_path = Path(test_folder.name)
+    for f in ("node_sets.json", "circuit_config.json", "input.dat"):
+        shutil.copy(SIM_DIR / f, test_folder_path)
+    with open(test_folder_path / CONFIG_FILE_MINI, "w") as f:
+        json.dump(sim_config_data, f, indent=2)
+
+    subprocess.run(
```

Comment:
Do you think we can tests without subprocess, but instead create a `Neurodamus` instance and running?

Reply:
Sure, I could try that. I just followed the format of the existing ones.

Reply:
I guess you mean something like:
`nd = Neurodamus(tmp_file.name, output_path=output_dir)`

And then instead of --output-path you test the option as a parameter to Neurodamus()?